packages/preload/src/index.ts

```diff
@@ -94,12 +94,12 @@ import type {
 } from '../../main/src/plugin/dockerode/libpod-dockerode';
 import type { CatalogExtension } from '../../main/src/plugin/extensions-catalog/extensions-catalog-api';
 import type { FeaturedExtension } from '../../main/src/plugin/featured/featured-api';
+import type { ContextGeneralState, ResourceName } from '../../main/src/plugin/kubernetes/contexts-states';
```

Comment:
Since you created the file, can we move it to the `/@/api` ?

Reply:
done